pkg/controller/direct/parametermanager/parameter_mapper.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +generated:mapper
+// krm.group: parametermanager.cnrm.cloud.google.com
+// krm.version: v1alpha1
+// proto.service: google.cloud.parametermanager.v1
```

Comment:
The `ModelToProto` function is incomplete. It needs to map all the fields from the `spec` to the GCP proto message, including the `value` of the parameter.